packages/engine/Source/Core/IonGeocoderService.js

```diff
@@ -25,6 +61,12 @@ function IonGeocoderService(options) {
   Check.typeOf.object("options.scene", options.scene);
   //>>includeEnd('debug');
 
+  const geocodeProviderType = defaultValue(
+    options.geocodeProviderType,
+    IonGeocodeProviderType.DEFAULT,
+  );
+  validateIonGeocodeProviderType(geocodeProviderType);
```

Comment:
Since this function throws a developer error, we should surround it with the debug pragmas so it is stripped out in production builds.

```suggestion
  //>>includeStart('debug', pragmas.debug);
  validateIonGeocodeProviderType(geocodeProviderType);
  //>>includeEnd('debug');
```